scripts/vault-init.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sleep 10
+# shellcheck source=/dev/null
+source $1/.env
```

Comment:
where do you pass a variable to the script?

Reply:
In the Vagrantfile and in .travis.yml

Reply:
ah!, ok, missed the `:args` section :)